File: official/docs/node/v6/referral-customers/refund-by-payment-log.js
const EasyPostClient = require('@easypost/api');

const client = new EasyPostClient(process.env.EASYPOST_API_KEY);

(async () => {
  const refund = await client.BetaReferralCustomer.refundByPaymentLog('paylog_...');

  console.log(refund);
})();

File: official/docs/csharp/v6/referral-customers/refund-by-payment-log.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using EasyPost;
using Newtonsoft.Json;

namespace EasyPostExamples
{
    public class Examples
    {
        public static async Task Main()
        {
            var client = new EasyPost.Client(new EasyPost.ClientConfiguration("EASYPOST_API_KEY"));

            EasyPost.Parameters.ReferralCustomer.RefundByPaymentLog parameters = new()
            {
                PaymentLogId = "paylog_...",
            };

            EasyPost.Models.API.PaymentRefund refund = await client.Beta.ReferralCustomer.RefundByPaymentLog(parameters);

            Console.WriteLine(JsonConvert.SerializeObject(refund, Formatting.Indented));
        }
    }
}
